tools/CommandLineTools.hx

```diff
@@ -485,15 +485,14 @@ class CommandLineTools
 				}
 
 			case MAC:
-				// if (System.hostArchitecture == X64) {
-
-				untyped $loader.path = $array(path + "Mac64/", $loader.path);
```

Comment:
This used to run every time. Is it safe to replace it with an `if` block with no final `else`?

Reply:
I don't think there is much point in adding Mac64 for non x86_64 bit systems. The architecture will not match so the ndll will fail to load anyway, so either way the ndll fails to load. But if you think it's better, I can change it so that Mac64 is added by default.

Reply:
No, you have a point there. I guess I didn't think it all the way through.